### PR TITLE
Oc rebase to k8s 1.20.0-beta.2

### PR DESCRIPTION
**Pull request by ingvagabund**
Following steps in https://github.com/openshift/enhancements/blob/master/enhancements/oc/upgrade_kubernetes.md

### CODE REVIEW

**Comment by ingvagabund** [comment]
https://github.com/openshift/origin/pull/25704 to address failing tests in `e2e-agnostic-cmd`.

File: pkg/helpers/kubeconfig/loader.go [hunk 1]
```diff
@@ -2,25 +2,17 @@ package kubeconfig
...
 
 	kclientcmd "k8s.io/client-go/tools/clientcmd"
-	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
 func NewPathOptions(cmd *cobra.Command) *kclientcmd.PathOptions {
-	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, genericclioptions.OpenShiftKubeConfigFlagName))
-}
-
-func NewPathOptionsWithConfig(configPath string) *kclientcmd.PathOptions {
```
Comment (soltysh) [inline pkg/helpers/kubeconfig/loader.go:15]: I'm inclined to drop this file entirely. The only difference between this and `kclientcmd.NewDefaultPathOptions` is `LoadingRules.DoNotResolvePaths` but I'd try it, it shouldn't affect us.

Comment (soltysh) [reply pkg/helpers/kubeconfig/loader.go:15]: From what I see we use it in login, logout, project, request project and projects commands.

**Review by soltysh — CHANGES_REQUESTED** [review]
As a follow-up have a look at how much of `oc debug` can we squash into `kubectl debug`

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by soltysh** [comment]
/close 
in favor of #660

**Pull request closed without merging**